ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+function extract_oc(){
+    echo -e "Extracting oc\n"
+    local retry=5 
+    tmp_oc="/tmp/client"
+    mkdir -p ${tmp_oc}
+    while ! (env "NO_PROXY=*" "no_proxy=*" oc adm release extract -a "${CLUSTER_PROFILE_DIR}/pull-secret" --command=oc --to=${tmp_oc} ${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE});
+    do
+        echo >&2 "Failed to extract oc binary, retry..."
+        (( retry -= 1 ))
+        if (( retry < 0 )); then return 1; fi
+        sleep 60
+    done
+    mv ${tmp_oc}/oc /tmp -f
+    which oc
+    oc version --client
+}
+
+# Define the checkpoints/steps needed for the specific case
+function post-ocp-66839(){
+    if [[ "${ADDITIONAL_ENABLED_CAPABILITIES}" != "CSISnapshot" ]] || [[ "${BASELINE_CAPABILITY_SET}" != "None" ]]; then
```

Comment:
Looks like the test-case defines baseline as `None` only. Just curious why we have the additional cap CSISnapshot as a prerequisite.

Reply:
It's a requirement from installer by jianlin, so we cover the same profile(`aws-ipi-disconnected-sts-basecap-none-addcap-f28`) here.

Reply:
Hmm, `ADDITIONAL_ENABLED_CAPABILITIES` is set with different values in different versions, e.g: for 4.13 source version, the optional cap is CSISnapshot, but on 4.14 source cluster, it is CSISnapshot, and Machine-API.

Reply:
Then we could control it in upgrade [config file](https://github.com/openshift/release/pull/43227/files#diff-48817126d567bdb6f9b42dc07689ff7d2a387dd3eb9eccc516448f7792282e10L201) in future versions, right? I think variable additional caps is not the checkpoint for this case, so we could keep CSISnapshot only in the following upgrade config.

Reply:
For the `ADDITIONAL_ENABLED_CAPABILITIES`, jinyum did a lot of work to cover all kinds of combination, the additional cap can be set via ENV vars, also some random selection via a dedicated step. Whatever the additional cap is set from where, the final setting would be landed into install-config.yaml, maybe we can get the additional setting from https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh#L143 to be more dynamic?

Reply:
And actually `CSISnapshot` setting on installation at the beginning has some configuration concerns, because storage operator is disabled, but  CSISnapshot actually can not work, we added it just because we want to ensure the additional cap can cause some difference on credential CR manifest for our case testing purpose. In the future, especially in next version, e.g: 4.14, more operators become optional, maybe we consider to change the setting. So I think we can discuss more about how to make the `ADDITIONAL_ENABLED_CAPABILITIES` judgement to be more reasonable.

Reply:
Hmm. yes, i know we have supported all kinds of combination for caps. And there are other cvo/upgrade cases covering them. For this case, the checkpoint is for `--include` cmd instead of optional caps, and so we drop the idea to integrate the checkpoint into existed upgrade case for optional caps in our case review stage, but add one more dedicated case is to cover it in upgrade process.
Jinyun's preface work could be used in optional caps cases integration, but it's not quite the same here, because not all caps have corresponding CRs, that means we need maintain another map for it. But actually it's not quite valuable because if `--include` cmd works in one combination, it would work in other combination. As for the command itself, no difference.

Reply:
FYI https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitems/testcase?query=trello%3AOTA%5C-555%20AND%20subteam.KEY%3Aota%20AND%20NOT%20status%3Ainactive 
Just checked there were 8 upgrade cases about optional caps, so above combination will be covered in those cases. As for `CSISnapshot`, if it makes us confused, i could remove it. Actually case ocp-66839 only cares  BASELINE_CAPABILITY_SET be None, due to no extra value, so I keep it there.

Reply:
Drop `ADDITIONAL_ENABLED_CAPABILITIES` to match the case checkpoint correctly.